app/javascript/react/assets/styles/colors.ts

```diff
@@ -1,17 +1,17 @@
 const cta = "#ffe651";
 const theme = "#00b2ef";
 
-const white = "#fff";
-const grey100 = "#f4f6f9";
+const white = "#Fff";
+const grey100 = "#F4f6f9";
 const grey200 = "#C0C9D1";
 const grey300 = "#7D858C";
 const grey400 = "#3E4449";
-const mint = "#81dbcb";
-const blue = "#00b3ef";
-const red = "#ff464c";
-const orange = "#f99435";
-const yellow = "#fdd44f";
-const green = "#86d175";
```

Comment:
I'm wondering - is there a specific reason why you changed that? 
I don't mind but I believe such changes can cause rebase/merge conflicts in the future, little but annoying to solve 😅